### PR TITLE
JSON is now converted to a Message containing the content_type of app…

### PR DESCRIPTION
**Pull request by SteveScorfield**
…lication/json

### CODE REVIEW

File: src/main/java/uk/gov/ons/census/fwmt/rmadapter/message/GatewayActionProducer.java [hunk 1]
```diff
@@ -38,9 +42,13 @@
 
   @Retryable
   public void sendMessage(Object dto) throws GatewayException {
+    Message gatewayMessage;
```
Comment (sdiaz1000) [inline src/main/java/uk/gov/ons/census/fwmt/rmadapter/message/GatewayActionProducer.java]: Java code style is declare where is created, so move this to line 48 so it reads 
Message gatewayMessage = convertJSONToMessage(JSONJobRequest);

File: src/main/java/uk/gov/ons/census/fwmt/rmadapter/message/GatewayActionProducer.java [hunk 2]
```diff
@@ -70,4 +78,13 @@ private String getCaseId(Object dto) {
...
     return caseId;
   }
+
+  public Message convertJSONToMessage(String messageJSON) {
+    Message gatewayMessage;
+    MessageProperties messageProperties = new MessageProperties();
+    messageProperties.setContentType("application/json");
+    MessageConverter messageConverter = new Jackson2JsonMessageConverter();
+
+    return gatewayMessage = messageConverter.toMessage(messageJSON, messageProperties);
```
Comment (sdiaz1000) [inline src/main/java/uk/gov/ons/census/fwmt/rmadapter/message/GatewayActionProducer.java]: maybe just get rid 
Message gatewayMessage;

and return
return  messageConverter.toMessage(messageJSON, messageProperties);

**Review by sdiaz1000 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/ons/census/fwmt/rmadapter/message/GatewayActionProducer.java (hunk 1 above)
Comment (SteveScorfield) [reply src/main/java/uk/gov/ons/census/fwmt/rmadapter/message/GatewayActionProducer.java]: Changed code.

File: src/main/java/uk/gov/ons/census/fwmt/rmadapter/message/GatewayActionProducer.java (hunk 2 above)
Comment (SteveScorfield) [reply src/main/java/uk/gov/ons/census/fwmt/rmadapter/message/GatewayActionProducer.java]: Have updated the code.

**Pull request merged**